src/IW/Core/Issue.hs

```diff
@@ -17,7 +18,14 @@ data Issue = Issue
     , issueBody      :: Text
     , issueRepoOwner :: RepoOwner
     , issueRepoName  :: RepoName
-    , issueUrl       :: Text
     , issueLabels    :: SqlArray Text
     } deriving stock (Generic, Show, Eq)
       deriving anyclass (ToJSON, FromRow, ToRow)
+
+issueUrl :: Issue -> Text
+issueUrl Issue{..} = "https://github.com/" 
+                  <> unRepoOwner issueRepoOwner 
```

Comment:
Let's align this pure function a little bit differently:

```haskell
issueUrl Issue{..} =
    "https://github.com/"
    <> unRepoOwner issueRepoOwner
    <> ...
```

In general we don't want to have big indentation of the following lines. So, for example, in this case to look code nicer you need to start from the line after `=`. But, for example, if you pattern-matching immediately, you can continue with the line with `=`

```haskell
issueUrl Issue{..} = case smth of  -- this is okay
    ....
    ....
```